manifests/application.pp

```diff
@@ -71,6 +72,9 @@
   }
 
   if $manage_db {
+    package { $postgresql_debversion_package:
+      ensure => installed,
+    }
```

Comment:
I think this should only be done if `$enable_deb` is true, right? Or will it be needed unconditionally?

Reply:
I think we need it unconditionally, because the database-migration in katello will also run unconditionally.

Reply:
yes, whether a content type is enabled/installed doesn't change the schema that is deployed, so its sitll needed